packages/zowe-explorer/src/Profiles.ts

```diff
@@ -626,7 +626,7 @@ export class Profiles extends ProfilesCache {
         return profileType;
     }
 
-    public async createZoweSchema(_zoweFileProvider: IZoweTree<IZoweTreeNode>): Promise<string> {
+    public async createZoweSchema(_zoweFileProvider: IZoweTree<IZoweTreeNode>): Promise<string | undefined> {
```

Comment:
Hey @sosadchuk, thanks for the contribution.  Do you have a scenario in mind that would return undefined?

Reply:
There are a couple other places in the function that simply call `return;` without any value specified.
I looked into this and it turns out that since `noImplicitReturns` is not enabled in the TypeScript config, it has omitted quite a few of these references. I filed issue #2850 to track resolving these implicit returns.